.ci/deps.sh

```diff
@@ -161,8 +161,8 @@ if [ ! -e ~/infer-linux64-v0.7.0/infer/bin ]; then
 fi
 
 # PMD commands
-if [ ! -e ~/pmd-bin-5.4.1/bin ]; then
-  wget -nc -O ~/pmd.zip https://github.com/pmd/pmd/releases/download/pmd_releases%2F5.4.1/pmd-bin-5.4.1.zip
+if [ ! -e ~/pmd-bin-5.6.1/bin ]; then
```

Comment:
Is this version upgrade essential for the fix? If so,t he bear version req should be bumped. If not, it should be a separate commit.